inst/examples/15-openapi-spec/entrypoint.R

```diff
@@ -19,34 +19,6 @@ handler <- function(num) { sum(as.integer(num)) }
 
 pr$handle("GET", "/sum", handler, serializer = serializer_json())
 
-# pr$run(swagger = swagger) # TODO-barret make function for this
+pr$setApiHandler(api_fun = openapi_func)
```

Comment:
I'd like to see the naming of this method share a "basename" with `$apiSpec()`...perhaps this should be named `$apiSpecTransform()`/`$apiSpecMutate()`?

Reply:
@meztez We still prefer `$apiSpecTransform()` over `$apiSpecHandler()` since `$handler()` means something different.